### PR TITLE
[0.0.4] Upgrade to kamon 0.6.1 and make kamon modules actually work

### PR DESCRIPTION
**Pull request by jypma**
Confirmed working in the document-core "reaktive" branch.

### CODE REVIEW

**Comment by jypma** [comment]
@alar17 and @vlezhnin please review

File: ts-reaktive-kamon-akka-cluster/src/main/scala/akka/kamon/cluster/AkkaClusterMonitoring.scala [hunk 1]
```diff
@@ -0,0 +1,67 @@
+package akka.kamon.cluster
+
+import akka.actor.ExtensionId
```
Comment (vlezhnin) [inline ts-reaktive-kamon-akka-cluster/src/main/scala/akka/kamon/cluster/AkkaClusterMonitoring.scala]: unused imports

File: ts-reaktive-kamon-akka/src/main/scala/akka/kamon/reaktive/AkkaSystemMonitoring.scala [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package akka.kamon.reaktive
+
+import scala.concurrent.duration._
```
Comment (vlezhnin) [inline ts-reaktive-kamon-akka/src/main/scala/akka/kamon/reaktive/AkkaSystemMonitoring.scala]: unused import

**Comment by jypma** [comment]
Removed unused imports. Please re-check.

**Comment by vlezhnin** [comment]
no more comments from me

**Pull request merged**